src/main/java/com/faforever/client/chat/ChannelTabController.java

```diff
@@ -293,6 +322,40 @@ protected String getInlineStyle(String username) {
     }
   }
 
+  public void onChangeTopicTextButtonClicked() {
+    String topic = chatChannel.getTopic();
+    topicTextField.setVisible(true);
+    topicTextField.setText(topic != null ? topic : "");
+    topicTextField.requestFocus();
+    topicTextField.selectEnd();
+  }
+
+  public void onAcceptTopicTextButtonClicked() {
+    String normalizedText = StringUtils.normalizeSpace(topicTextField.getText());
+    if (!normalizedText.equals(chatChannel.getTopic())) {
+      topicPane.setDisable(true);
+      topicTriggeredByUser = true;
+      chatService.setChannelTopic(channelName, normalizedText);
+    } else {
+      topicTextField.setVisible(false);
+    }
+  }
+
+  public void onCancelChangesTopicTextButtonClicked() {
+    String topic = chatChannel.getTopic();
+    topicTextField.setText(topic != null ? topic : "");
+    topicTextField.setVisible(false);
+    topicPane.setDisable(false);
+  }
+
+  public void onAcceptChangesTopicTextField() {
+    onAcceptTopicTextButtonClicked();
+  }
```

Comment:
What is the purpose of this method?

Reply:
It's the second way to accept the changes via the `Enter` button.

Reply:
Why not just use the onAcceptTopicTextButtonClicked there then? It seems a little superfluous to have a method that just directly calls another method with the exact same arguments